### PR TITLE
feat: redesign and refactor mobile sticky monthly good selector

### PR DESCRIPTION
**Pull request by eddieferrer**
Screen Shot 2021-08-05 at 1 47 27 PM

- Redesigned the flow of the sticky monthly good selector on mobile. 
- Refactored to use kv-ui elements and tailwinds

TODO:
- Copy should come from contentful

Reference figma: 
https://www.figma.com/file/weQAkMDYoBA5ZnuHf4TPOA/Cause-Based-Landing-Page-Templates?node-id=136%3A1442

Minor differences between the figma and implementation to use our kv-ui-elements kvlightbox standards. (button position, title position, height of lightbox on mobile, etc)

SUBS-783

### CODE REVIEW

File: src/components/MonthlyGood/MonthlyGoodSelectorMobile.vue [hunk 1]
```diff
@@ -1,26 +1,46 @@
...
-					]"
-				>
-					Get started
-				</kv-button>
-			</div>
-		</div>
+	<div>
+		<kv-button
+			class="tw-mx-auto tw-w-full"
+			@click.native="showLightbox"
```
Comment (ryan-ludwig) [inline src/components/MonthlyGood/MonthlyGoodSelectorMobile.vue:5]: minor, the `kv-ui-elements/kv-button` doesn't need the `.native` anymore

File: src/components/MonthlyGood/MonthlyGoodSelectorMobile.vue [hunk 2]
```diff
@@ -29,58 +49,55 @@
...
-					v-for="(option, index) in mgAmountOptions"
-					:key="index"
-					v-kv-track-event="[
-						'homepage',
-						'click-mgpromo-amount',
-						option.value
-					]"
-					@click="selectAmount(option.value)"
+			<div class="tw-mt-2" v-if="lightboxStep === 'amount'">
+				<h4 class="tw-py-1">
```
Comment (ryan-ludwig) [inline src/components/MonthlyGood/MonthlyGoodSelectorMobile.vue:63]: This should probably be `<label class="tw-text-h4 tw-py-1" for="mgAmountDropdown">` so the select has a label

File: src/components/MonthlyGood/MonthlyGoodSelectorMobile.vue [hunk 3]
```diff
@@ -216,84 +239,10 @@ export default {
...
-			return this.lightboxStep === 'cause'
-				? 'What cause would you like to support?' : 'How much would you like to lend?';
-		}
 	}
 };
 
 </script>
 
 <style lang="scss" scoped>
-@import 'settings';
```
Comment (ryan-ludwig) [inline src/components/MonthlyGood/MonthlyGoodSelectorMobile.vue:229]: niiiice

**Review by ryan-ludwig — APPROVED** [review]
Minor thing about the <label>, otherwise looks great!

**Review by BoulderBrains — COMMENTED** [review]
👀

**Pull request merged**